djconnectwise/sync.py

```diff
@@ -408,9 +408,9 @@ def _assign_field_data(self, instance, json_data):
                 '- skipping.'.format(instance.id)
             )
         except ObjectDoesNotExist as e:
-            logger.warning(
-                'Service note {} has a ticketId that does not exist. {}'.
-                format(instance.id, e)
+            raise InvalidObjectException(
```

Comment:
I removed the logger warning because only logging and not raising an exception caused an issue in #653. Not sure if this is makes sense here though.